server/src/constants/default-payload.ts

```diff
@@ -22,3 +24,14 @@ export const DEFAULT_CUSTOMER_PAYLOAD = {
   contactNumber: '',
   address: '',
 };
+
+/**
+ * Contains the default values for the fields on a ListingPayload object which
+ * are automatically set by ListingService during listing creation.
+ */
+export const DEFAULT_LISTING_PAYLOAD: Required<ListingComputedProperties> = {
```

Comment:
instead of creating this constant, can datastore support default value if not set?

Reply:
So far I haven't found anything that suggests that it supports default value yet. But I'm not too sure about that.
But either way, in our case, I prefer setting the values ourselves. For example, fields like listingStatus must have value 'ongoing'. I don't want to let the client provide another value for this field at the point of creation.